crates/ide/src/syntax_highlighting.rs

```diff
@@ -298,8 +299,9 @@ fn traverse(
                         ast::Item::Fn(_) | ast::Item::Const(_) | ast::Item::Static(_) => {
                             bindings_shadow_count.clear()
                         }
-                        ast::Item::MacroCall(_) => {
+                        ast::Item::MacroCall(ref macro_call) => {
                             inside_macro_call = true;
+                            inside_proc_macro_call = sema.is_proc_macro_call(macro_call);
```

Comment:
```suggestion
                        ast::Item::MacroCall(macro_call) => {
                            inside_macro_call = true;
                            inside_proc_macro_call = sema.is_proc_macro_call(&macro_call);
```

Reply:
The borrow checker grumbles here 😅 

```
error[E0382]: borrow of partially moved value: `item`
   --> crates/ide/src/syntax_highlighting.rs:310:52
    |
287 |                 if let Some(item) = ast::Item::cast(node.clone()) {
    |                             ---- this reinitialization might get skipped
...
302 |                         ast::Item::MacroCall(macro_call) => {
    |                                              ---------- value partially moved here
...
310 |                         if sema.is_attr_macro_call(&item) {
    |                                                    ^^^^^ value borrowed here after partial move
    |
    = note: partial move occurs because value has type `syntax::ast::MacroCall`, which does not implement the `Copy` trait
help: borrow this binding in the pattern to avoid moving the value
    |
302 |                         ast::Item::MacroCall(ref macro_call) => {
    |                                              +++

```

Reply:
Oh, I didn't notice the `continue`s, nevermind then.